src/help/components/contextual/SideButtons.tsx

```diff
@@ -16,6 +17,36 @@ const SideButtons = ({ displayHelp, onClick }: Props) => {
   const [displayFeedback, setDisplayFeedback] = useState(false);
 
   useEffect(() => {
+    // Checking if there is a scroll bar
+    const mainContent = document?.querySelector<HTMLElement>(
+      `.${baseStyles['main-content']}`
+    );
+    const scrollBarWidth =
+      mainContent && mainContent.offsetWidth - mainContent.clientWidth;
+
+    if (scrollBarWidth) {
+      const hjButton = document.querySelector<HTMLElement>(
+        '#_hj_feedback_container div button'
+      );
+
+      const sideButton = document?.querySelector<HTMLElement>(
+        `.${sideButtonStyles['side-button']}`
+      );
+      const sideButtonHelp = document?.querySelector<HTMLElement>(
+        `.${sideButtonStyles.help}`
+      );
+
+      if (hjButton) {
+        hjButton.style.right = '17px';
```

Comment:
Shouldn't this, and the other "right" values below, be "= `${scrollBarWidth}px`"?

Reply:
On macs, the scroll bar width calculated is 15px, but that still has some overlap with the scroll bar for some reason. 15px looks like the right-most edge of the button on actual scrolling part of the scroll bar, while with 17px the edge is on the scroll area (if that makes sense). 

Windows has 17px regardless